rust/src/combinators/measure_cast/bounded_range_to_zCDP/mod.rs

```diff
@@ -0,0 +1,50 @@
+use crate::{
+    core::{Domain, Measurement, Metric, MetricSpace, PrivacyMap},
+    error::Fallible,
+    measures::{BoundedRange, ZeroConcentratedDivergence},
+    traits::Float,
+};
+
+/// Constructs a new output measurement where the output measure
+/// is casted from `MaxDivergence<QO>` to `BoundedRange<QO>`.
+///
+/// # Arguments
+/// * `meas` - a measurement with a privacy measure to be casted
+///
+/// # Generics
+/// * `DI` - Input Domain
+/// * `DO` - Output Domain
+/// * `MI` - Input Metric
+/// * `QO` - Output distance type. One of `f32` or `f64`.
+/// For more details, see: https://differentialprivacy.org/exponential-mechanism-bounded-range/
+
+pub fn make_bounded_range_to_zCDP<DI, TO, MI, QO>(
+    m: Measurement<DI, TO, MI, BoundedRange<QO>>,
+) -> Fallible<Measurement<DI, TO, MI, ZeroConcentratedDivergence<QO>>>
+where
+    DI: Domain,
+    MI: 'static + Metric,
+    QO: Float,
+    (DI, MI): MetricSpace,
+{
+    let privacy_map: PrivacyMap<MI, BoundedRange<QO>> = m.privacy_map.clone();
+    // For Mike: How to make this sane? I just want an 8
+    let constant: QO = QO::one()
+        + QO::one()
+        + QO::one()
+        + QO::one()
+        + QO::one()
+        + QO::one()
+        + QO::one()
+        + QO::one();
```

Comment:
Now that QO is just f64 this is simple, but `QO::exact_int_cast(8)?` would have worked.